packages/pnpm/src/cmd/outdated.ts

```diff
@@ -88,42 +91,72 @@ export default async function (
 
   if (!outdatedPackages.length) return
 
-  let columnNames = [
-    'Package',
-    'Current',
-    'Latest'
-  ]
+  if (opts.table !== false) {
+    let columnNames = [
+      'Package',
+      'Current',
+      'Latest'
+    ]
 
-  if (opts.long) {
-    columnNames.push('Details')
-  }
+    if (opts.long) {
+      columnNames.push('Details')
+    }
 
-  columnNames = columnNames.map((name: string) => chalk.blueBright(name))
-  let columnFns: Array<(outdatedPkg: OutdatedWithVersionDiff) => string> = [
-    renderPackageName,
-    renderCurrent,
-    renderLatest,
-  ]
+    columnNames = columnNames.map((name: string) => chalk.blueBright(name))
+    let columnFns: Array<(outdatedPkg: OutdatedWithVersionDiff) => string> = [
+      renderPackageName,
+      renderCurrent,
+      renderLatest,
+    ]
+
+    if (opts.long) {
+      columnFns.push(renderDetails)
+    }
+
+    return table([
+      columnNames,
+      ...sortedPackages(outdatedPackages).map((outdatedPkg) => columnFns.map((fn) => fn(outdatedPkg))),
+    ], TABLE_OPTIONS)
+  } else {
+    return sortedPackages(outdatedPackages)
+      .map((outdatedPkg) => {
+        let info = stripIndent`
+          ${renderPackageName(outdatedPkg)}
+          ${renderCurrent(outdatedPkg)} => ${renderLatest(outdatedPkg)}`
```

Comment:
Make arrow grey: `${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`

(so that it doesn't add visual noise when reading the version numbers)